imm_ptl_core/src/main/java/qouteall/imm_ptl/core/compat/mixin/MixinIrisIris.java

```diff
@@ -2,6 +2,7 @@
 
 import net.coderbot.iris.Iris;
 import net.coderbot.iris.shaderpack.DimensionId;
```

Comment:
If this import is no longer required it should be removed.

Reply:
All imports on this file are still in use. `cir.setReturnValue(DimensionId.OVERWORLD)` and `@Mixin(value = Iris.class, remap = false)`

Reply:
Oh, I missed that setReturnValue. Currently without an IDE :sweat_smile: That's fine then :+1: